src/piaf/templates/piaf/admin.html

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
This is a light admin for uploading a JSON file with texts and soon downloading the resulting JSON with annotations.

Reply:
The file you mention is not VALID JSON and does not have the required data:
```
{"text": "Les activit\u00e9s agricoles x…"}
{"text": "C'est l'une des solutions propos\u00e9es et recommand\u00e9es par l'ONU…"}
```

The expected file should look like:
```json
{"data": [
  {"title": "Article 2", "paragraphs": [
    {"context": "Paragraph 1"}, 
    {"context": "Paragraph 2"}
  ]},
  {"title": "Article 2", "paragraphs": [
    {"context": "Paragraph 1"}, 
    {"context": "Paragraph 2"},
    {"context": "Paragraph 3"}
  ]}
]}
```

Reply:
Exact. Well spotted. I uploaded a file corresponding to this proper format. It is well uploaded (As far I can see in the database). However, No connexion between the API and the frontend yet. That's normal right ?

![Screen Shot 2019-10-07 at 16 10 21](https://user-images.githubusercontent.com/13087916/66319222-080e7780-e91d-11e9-86b4-b5852cd8f6ae.png)